### PR TITLE
Homeworks

### PR DESCRIPTION
**Pull request by cmoreno12**

### CODE REVIEW

**Comment by cmoreno12** [comment]
Cyclomatic Complexity Homework

File: bogota-abril-2019/css-magazine-cover/Pagina.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+			<p>www.vibe.ng</p>
+
+			<h1>vibe<span>.ng</span></h1>
+			<h2>the rise & rise of<span>ycee</span></h2>
+		</header>
+
+		<article>
+			<section>
+				<h1>+</h1>
+				<h2>3 nigerian fasion designers</h2>
```
Comment (gabrielmartinezglobant) [inline bogota-abril-2019/css-magazine-cover/Pagina.html]: Seems All this text is in the same context. Please review the use of the semantic.

File: bogota-abril-2019/cyclomatic-complexity/agePredictor.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+
+//la complejidad del algoritmo es 2, se puede encontrar adjunta en el archivo cyclomatic complexity.jpg
+function predictAge(age1, age2, age3, age4, age5, age6, age7, age8) {
+    var ages = [...arguments];
```
Comment (gabrielmartinezglobant) [inline bogota-abril-2019/cyclomatic-complexity/agePredictor.js]: Nice aproach the spread over arguments.
But if do so, unnecesary use of explicit use of argument in the function declaration

Comment (cmoreno12) [reply bogota-abril-2019/cyclomatic-complexity/agePredictor.js]: ok, if I understand, the correct use of arguments should be like this (¿?):

function predictAge(...arguments){

**Pull request closed without merging**